README.md

```diff
@@ -57,6 +58,24 @@ To completely uninstall the Drupal CMS Launcher and remove all associated files:
 
 After these steps, the application and all its data will be fully removed from your system.
 
+## Troubleshooting
+
+This section provides solutions to common problems.
+
+### Windows Defender Firewall is Blocking the Application
+
+On Windows, the built-in firewall may block the launcher because it needs to start a local web server. If the application fails to start or gives a connection error, you may need to manually allow it through the firewall.
+
+**Solution:**
+
+1.  Press the Windows Key, type `Allow an app through Windows Firewall`, and open it.
+2.  Click the **"Change settings"** button. You may need administrator permission.
+3.  Click the **"Allow another app..."** button at the bottom.
+4.  In the new window, click **"Browse..."** and navigate to where you ran the `cms-launcher`. Select the main executable file and click **"Open"**.
```

Comment:
```suggestion
4.  In the new window, click **"Browse..."** and navigate to where you installed the launcher (usually SOME_VALUE_HERE). Select `Launch Drupal CMS.exe` and click **"Open"**.
```

The launcher uses the NSIS one-click installer, so it should generally go to a standard path. I'd prefer if we provided the example paths here to help guide folks.